test/fixtures/output/connect/plugin/install/platform-not-found.golden

```diff
@@ -1,3 +1,4 @@
+Beginning installation.
```

Comment:
How long does installation take? If < 1s, I think we should remove this message. Our other commands don't do this.

Of course, you can always log it with DEBUG!

Reply:
I frankly just did it so that the spacing would work out across all the different logic branches, but I just removed it and wrestled with the spacing for a bit.